api/v1alpha1/olsconfig_types.go

```diff
@@ -401,6 +407,75 @@ type ProxyConfig struct {
 	ProxyCACertificateRef *corev1.LocalObjectReference `json:"proxyCACertificate,omitempty"`
 }
 
+// MCPServer defines the settings for a single MCP server.
+type MCPServer struct {
+	// Name of the MCP server
+	// +kubebuilder:validation:Required
+	// +required
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Name"
+	Name string `json:"name,omitempty"`
+	// Transport of the MCP server
+	// it should be one of the following: sse, stdio, streamable_http
+	// +kubebuilder:validation:Enum=sse;stdio;streamable_http
```

Comment:
can drop sse from this list now?

Reply:
In the structure MCPServerStreamableHTTPTransport I put a option to activate SSE support:
https://github.com/openshift/lightspeed-operator/pull/894/files/f536311765d96191d7ccb9f823e92aeb9dbb1446#diff-77e6d7fe1846c7ad02af9efceb28f7be71b1335993f6d36664418fb4c3bec33bR476
If user like to use SSE, turn that option on, we save the SSE field here.